library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -120,7 +119,6 @@ class Client() {
         }
     }
 
-    @WorkerThread
```

Comment:
Had to remove these annotations since they were the only thing that was requiring importing the entire androidx library.